### PR TITLE
Modernize table kit

### PR DESCRIPTION
**Pull request by eabousaif**
Modernize the table playbook kit

### CODE REVIEW

File: app/pb_kits/playbook/pb_table/table.rb [hunk 1]
```diff
@@ -2,94 +2,48 @@
...
+                  values: %w[sm md lg],
+                  default: "md"
+      prop :single_line, type: Playbook::Props::Boolean,
+                         default: false
+      prop :dark, type: Playbook::Props::Boolean,
+                  default: false
+      prop :disable_hover, type: Playbook::Props::Boolean,
+                           default: false
+      prop :container, type: Playbook::Props::Boolean,
+                       default: false
```
Comment (garettarrowood) [inline app/pb_kits/playbook/pb_table/table.rb]: Every story that uses the table kit that we bring in front of Kraig comes back to us with him wanting `container: true`. Can we set this as the default option, @jasoncypret ?

**Comment by garettarrowood** [comment]
Nice work on this conversion!

File: app/pb_kits/playbook/pb_table/table.rb (hunk 1 above)
Comment (eabousaif) [reply app/pb_kits/playbook/pb_table/table.rb]: As per the last conversation with Jason, the default for container has been set to true.

File: app/pb_kits/playbook/pb_table/docs/_table_sm.html.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<%= pb_rails("table", props: {size: "sm"}) do %>
+<%= pb_rails("table", props: { size: "sm", container: false }) do %>
```
Comment (darrendc) [inline app/pb_kits/playbook/pb_table/docs/_table_sm.html.erb]: It's picky, but you used a space after opening and before closing the curly brackets here, but did not do that in the other .erb files.

Comment (eabousaif) [reply app/pb_kits/playbook/pb_table/docs/_table_sm.html.erb]: Fixed the spacing in all html files, thanks!

**Pull request merged**